yucca/training/data_loading/YuccaDataModule.py

```diff
@@ -11,9 +12,10 @@ class YuccaDataModule(pl.LightningDataModule):
     def __init__(
         self,
         configurator: YuccaConfigurator,
-        pred_data_dir: str = None,
         composed_train_transforms: torchvision.transforms.Compose = None,
         composed_val_transforms: torchvision.transforms.Compose = None,
+        num_workers: int = 8,
+        pred_data_dir: str = None,
```

Comment:
I believe this will throw and error, and is thus not a very good default value. To me when a parameter has a default parameter it means "you can change this _if you want_", which is not true here. I would either require the parameter or choose a default param which does not result in a crash.